shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/java/org/apache/shardingsphere/sql/parser/oracle/visitor/impl/OracleDMLVisitor.java

```diff
@@ -474,4 +474,9 @@ public ASTNode visitGroupByClause(final GroupByClauseContext ctx) {
         }
         return new GroupBySegment(ctx.getStart().getStartIndex(), ctx.getStop().getStopIndex(), items);
     }
+    
+    @Override
+    public ASTNode visitSubquery(final OracleStatementParser.SubqueryContext ctx) {
```

Comment:
Is it possible to use `SubqueryContext` instead of `OracleStatementParser.SubqueryContext`

Reply:
Ok.